packages/fuels-types/src/utils.rs

```diff
@@ -1,69 +0,0 @@
-use lazy_static::lazy_static;
```

Comment:
So IIUC this is leftover from the `fuel-abi-type` repo migration?

Reply:
That was moved to the `fuel-abi-type` repo recently.